core/WFCORE-6960_Simple_config_export_for_a_server_as_an_attachment_for_standalone_or_domain_mode.adoc

```diff
@@ -0,0 +1,143 @@
+---
+categories:
+ - core
+ - management
+---
+= [Community] Simple config export for a server as an attachment for standalone or domain mode 
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          management,core,configuration
+
+== Overview
+
+The purpose of this feature is to enrich the management model with a new operation *read-config-as-xml-file* equivalent to the existing *read-config-as-xml" that would return a stream to be either displayed or saved to a file.
```

Comment:
Nitpicking I know, but there's a minor typo here: s/`*read-config-as-xml"`/`*read-config-as-xml*`/

Reply:
Thanks, can be resolved.